torch/_library/custom_ops.py

```diff
@@ -763,6 +765,89 @@ def wrapped_func(keyset, *args, **kwargs):
         else:
             return register(func)
 
+    def register_autocast(
+        self,
+        device_type: str,
+        cast_inputs: _dtype,
+    ):
```

Comment:
Can you check that calling custom_opregister_autocast multiple times does not error out? That's one of the feature of the torch.library.custom_op api